Makefile

```diff
@@ -146,8 +146,13 @@ setup/travis:
 	@echo Booting Minikube up, see Travis env. variables for more information
 	@mkdir -p $HOME/.kube $HOME/.minikube
 	@touch $KUBECONFIG
-	@sudo minikube start --vm-driver=none --kubernetes-version=v1.16.0
+	@sudo minikube start --vm-driver=none
 	@sudo chown -R travis: /home/travis/.minikube/
+	sudo ./hack/modify_etc_hosts.sh "keycloak.local"
+	@sudo minikube addons enable ingress
+	@sudo minikube addons enable ingress-dns
```

Comment:
If we want to remove configuration of `/etc/hosts` and use ingress-dns instead, we probably need to add `minikube ip` as a DNS on machine it is running on. More info [here](https://github.com/kubernetes/minikube/tree/master/deploy/addons/ingress-dns#solution).

Reply:
Done!.